snorkel/labeling/model/logger.py

```diff
@@ -0,0 +1,49 @@
+from collections import defaultdict
+
+
+class Logger:
```

Comment:
This is fine for this PR, since it's just porting. But we should replace this with the LogManager (identifies when it's time to log) and LogWriter (writes logs to file or tensorboard), since they do the same thing.

Reply:
Noted. cc @ajratner @paroma for new label model effort